src/components/TodayWeather/TodayWeather.jsx

```diff
@@ -0,0 +1,69 @@
+import PropTypes from "prop-types";
+import Pin from "../../assets/icons/pin.png";
+import Crosshair from "../../assets/icons/crosshair.png";
+import RoundButton from "../../components/RoundButton/RoundButton";
+import { UNITS, ICON_URL } from "../../constants";
+import "./TodayWeather.css";
+
+const TodayWeather = ({
+  handleLocationSearchVisibility,
+  cityName,
+  temperature,
+  description,
+  image,
+  unit,
+  handleOnClick,
```

Comment:
Values first, methods in the end:

```suggestion
  cityName,
  temperature,
  description,
  image,
  unit,
  handleLocationSearchVisibility,
  handleGeolocationClick,
```